index.js

```diff
@@ -55,6 +55,7 @@ export default function (window) {
     }
     const view = newView(doc, options)
     const svg = view.render()
+    svg.classList.add(`scratchblocks-style-${options.style}`)
```

Comment:
Maybe we could add a lil comment explaining why we do this (to get the right colors for `scratch3-high-contrast`)?